momento-sdk/src/main/java/momento/sdk/TokenClient.java

```diff
@@ -0,0 +1,217 @@
+package momento.sdk;
+
+import com.google.gson.Gson;
+import com.google.protobuf.ByteString;
+import grpc.permission_messages.CacheRole;
+import grpc.permission_messages.ExplicitPermissions;
+import grpc.permission_messages.Permissions;
+import grpc.permission_messages.PermissionsType;
+import grpc.permission_messages.TopicRole;
+import java.util.concurrent.CompletableFuture;
+import javax.annotation.Nonnull;
+import momento.sdk.auth.CredentialProvider;
+import momento.sdk.auth.accessControl.CacheItemSelector;
+import momento.sdk.auth.accessControl.CacheSelector;
+import momento.sdk.auth.accessControl.DisposableToken;
+import momento.sdk.auth.accessControl.DisposableTokenPermission;
+import momento.sdk.auth.accessControl.DisposableTokenScope;
+import momento.sdk.auth.accessControl.ExpiresIn;
+import momento.sdk.auth.accessControl.TopicSelector;
+import momento.sdk.exceptions.InvalidArgumentException;
+import momento.sdk.exceptions.UnknownException;
+import momento.sdk.responses.GenerateDisposableTokenResponse;
+import momento.token._GenerateDisposableTokenRequest;
+import momento.token._GenerateDisposableTokenResponse;
+
+public class TokenClient {
```

Comment:
@nand4011 I think this is intended to be an internal class, and I'm trying to remember what our strategy was for making classes like this invisible / less visible to end users in the IDE? do we make them package-protected or something?

Reply:
yeah it looks like the pattern in this repo so far is to prefix them with `Scs` and make them package-protected so they aren't public outside of the source tree.